src/widgets/RainWidget.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import GenericWidget from './GenericWidget';
+import './RainWidget.css';
+
+class RainWidget extends Component {
+
+  render() {
+    return (
+      <GenericWidget class="RainWidget" icon="cloud"
+          isOn={this.props.rain > 0}
+          value1={<div className="rain">{this.props.rain}</div>}
+          value2={<div className="rate">{this.props.rate}</div>}
```

Comment:
Thanks, I indeed used the WattWidget method, but I used value1 in stead of h2, so it didn't work for me. But this works :)